Core/include/Acts/Geometry/GeometryIdentifier.hpp

```diff
@@ -53,6 +55,8 @@ class GeometryIdentifier {
   constexpr Value approach() const { return getBits(kApproachMask); }
   /// Return the sensitive identifier.
   constexpr Value sensitive() const { return getBits(kSensitiveMask); }
+  /// Return the extra identifier
+  constexpr Value extra() const { return getBits(kExtraMask); }
```

Comment:
should we state possible use cases somewhere?

one thing that comes to my mind is tagging surfaces of interest e.g. something in the endcaps that can easily be filtered while construction